Source/WebCore/platform/text/SegmentedString.h

```diff
@@ -25,10 +25,6 @@
 
 namespace WebCore {
 
-// FIXME: This should not start with "k".
```

Comment:
Seems to be a cleanup unrelated to this patch ?

Reply:
Was necessary to fix conflicting redefinitions that resulted from new imports of `CSSTokenizer.h`.